### PR TITLE
Renamed FetchRequests to Fetch

### PR DESCRIPTION
**Pull request by VeronikaSolovei9**
This PR is a part of future response fetcher integration. 
Initial code contains only a code related to `FetchRequest` function rename to `Fetch`. 
Do we want to remane package name as well? 
If yes - what name this should be? 
If yes - it has a dependency in configs, like `PBS_STORED_REQUESTS_HTTP_EVENTS_AMP_ENDPOINT: "serviceURL/prebid/stored-requests"` - do we want to rename this in configs? 
Do we also want to remane data structured in config.go?

### CODE REVIEW

File: stored_requests/fetcher_test.go [hunk 1]
```diff
@@ -42,14 +42,14 @@ func TestPerfectCache(t *testing.T) {
...
+	reqData, impData, errs := aFetcherWithCache.Fetch(ctx, reqIDs, impIDs)
 
 	reqCache.AssertExpectations(t)
 	impCache.AssertExpectations(t)
 	fetcher.AssertExpectations(t)
 	metricsEngine.AssertExpectations(t)
 	assert.JSONEq(t, `{"req":true}`, string(reqData["req-id"]), "Fetch requests should fetch the right request data")
-	assert.JSONEq(t, `{}`, string(impData["known"]), "FetchRequests should fetch the right imp data")
+	assert.JSONEq(t, `{}`, string(impData["known"]), "Fetch should fetch the right imp data")
 	assert.Len(t, errs, 0, "FetchRequest shouldn't return any errors")
```
Comment (guscarreon) [inline stored_requests/fetcher_test.go:53]: I believe we missed this line too

File: stored_requests/fetcher_test.go [hunk 2]
```diff
@@ -82,14 +82,14 @@ func TestImperfectCache(t *testing.T) {
...
 
 	impCache.AssertExpectations(t)
 	fetcher.AssertExpectations(t)
 	metricsEngine.AssertExpectations(t)
 	assert.Len(t, reqData, 0, "Fetch requests should return nil if no request IDs were passed")
-	assert.JSONEq(t, `true`, string(impData["cached"]), "FetchRequests should fetch the right imp data")
-	assert.JSONEq(t, `false`, string(impData["uncached"]), "FetchRequests should fetch the right imp data")
+	assert.JSONEq(t, `true`, string(impData["cached"]), "Fetch should fetch the right imp data")
+	assert.JSONEq(t, `false`, string(impData["uncached"]), "Fetch should fetch the right imp data")
 	assert.Len(t, errs, 0, "FetchRequest shouldn't return any errors")
```
Comment (guscarreon) [inline stored_requests/fetcher_test.go:93]: I believe we missed this line

**Review by guscarreon — COMMENTED** [review]
> If yes - what name this should be?

To me stored requests works. Thoughts?
> If yes - it has a dependency in configs, like PBS_STORED_REQUESTS_HTTP_EVENTS_AMP_ENDPOINT: "serviceURL/prebid/stored-requests" - do we want to rename this in configs?

I believe the name as is fine.
> Do we also want to remane data structured in config.go?

I believe we should leave it as is. Specially because changing say `stored_requests` would imply that many PBS users would have to modify their configuration files or environment variables before updating to the latest PBS.
```
 40    StoredRequests    StoredRequests  `mapstructure:"stored_requests"`
 41    StoredRequestsAMP StoredRequests  `mapstructure:"stored_amp_req"`
        .
        .
        .
690    v.SetDefault("stored_requests.filesystem.enabled", false)
691    v.SetDefault("stored_requests.filesystem.directorypath", "./stored_requests/data/by_id")
692    v.SetDefault("stored_requests.directorypath", "./stored_requests/data/by_id")
693    v.SetDefault("stored_requests.postgres.connection.dbname", "")
694    v.SetDefault("stored_requests.postgres.connection.host", "")
695    v.SetDefault("stored_requests.postgres.connection.port", 0)
696    v.SetDefault("stored_requests.postgres.connection.user", "")
697    v.SetDefault("stored_requests.postgres.connection.password", "")
698    v.SetDefault("stored_requests.postgres.fetcher.query", "")
699    v.SetDefault("stored_requests.postgres.fetcher.amp_query", "")
700    v.SetDefault("stored_requests.postgres.initialize_caches.timeout_ms", 0)
701    v.SetDefault("stored_requests.postgres.initialize_caches.query", "")
702    v.SetDefault("stored_requests.postgres.initialize_caches.amp_query", "")
703    v.SetDefault("stored_requests.postgres.poll_for_updates.refresh_rate_seconds", 0)
704    v.SetDefault("stored_requests.postgres.poll_for_updates.timeout_ms", 0)
705    v.SetDefault("stored_requests.postgres.poll_for_updates.query", "")
706    v.SetDefault("stored_requests.postgres.poll_for_updates.amp_query", "")
707    v.SetDefault("stored_requests.http.endpoint", "")
708    v.SetDefault("stored_requests.http.amp_endpoint", "")
709    v.SetDefault("stored_requests.in_memory_cache.type", "none")
710    v.SetDefault("stored_requests.in_memory_cache.ttl_seconds", 0)
711    v.SetDefault("stored_requests.in_memory_cache.request_cache_size_bytes", 0)
712    v.SetDefault("stored_requests.in_memory_cache.imp_cache_size_bytes", 0)
713    v.SetDefault("stored_requests.cache_events_api", false)
714    v.SetDefault("stored_requests.http_events.endpoint", "")
715    v.SetDefault("stored_requests.http_events.amp_endpoint", "")
716    v.SetDefault("stored_requests.http_events.refresh_rate_seconds", 0)
717    v.SetDefault("stored_requests.http_events.timeout_ms", 0)
config/config.go
```

File: stored_requests/fetcher_test.go (hunk 1 above)
Comment (VeronikaSolovei9) [reply stored_requests/fetcher_test.go]: Yes, updated

File: stored_requests/fetcher_test.go (hunk 2 above)
Comment (VeronikaSolovei9) [reply stored_requests/fetcher_test.go]: Updated

**Review by guscarreon — APPROVED** [review]
LGTM

**Comment by mansinahar** [comment]
Closing this in favor of #2099. Will reopen if needed.

**Pull request closed without merging**